curated_transformers/models/pytorch/hf_util.py

```diff
@@ -37,7 +42,11 @@ def convert_pretrained_model_for_encoder(
     elif isinstance(encoder, RobertaEncoder):
         converted = _convert_roberta_base_state(params)
     else:
-        raise ValueError(f"Unsupported encoder type: {type(encoder)}")
+        # We don't need to handle the exceptional case
+        # here as the relevant checks are already performed
+        # during the construction of the CuratedTransformer
+        # instance.
+        pass
```

Comment:
I think we should have the exception here. Would be bad to silently do nothing if the user triggered this code path somehow.

Reply:
Fair. I guess I'm just peeved a bit that we essentialy have to maintain the supported list of encoders in two different modules, but I don't see any way around it 🤷